lib/v8debugapi.js

```diff
@@ -331,6 +331,14 @@ module.exports.create = function(logger_, config_, fileStats_) {
               uncompiled + ' <<', err);
           }
         };
+      case 'es6':
```

Comment:
How likely is it that the users are naming their es6 source files with these extensions instead of just `.js`? If I were to do a project with Babel, I would like my source files to have the `.js` suffix. But then again, I haven't done anything with Babel yet, so I don't know what the standard practice is.